docs/dev/Lua API.rst

```diff
@@ -5766,82 +5766,129 @@ The names of the functions are also available as the keys of the
 building-hacks
 ==============
 
-This plugin overwrites some methods in workshop df class so that mechanical workshops are possible. Although
-plugin export a function it's recommended to use lua decorated function.
+This plugin overwrites some methods in workshop df class to change and extend workshop
+functionality. See exported function list for details.
 
 .. contents::
   :local:
 
 Functions
 ---------
 
-``registerBuilding(table)`` where table must contain name, as a workshop raw name, the rest are optional:
+* ``setOwnableBuilding(workshop_type, enable)``
 
-    :name:
-        custom workshop id e.g. ``SOAPMAKER``
+  Set workshop to be included in zones (such as bedroom or inn).
```

Comment:
Why would you choose to include the workshop in zones? What does the choice affect?

Reply:
It allows for interesting systems: one example i had (before big gui changes etc...) was a personal dwarven hobby workshop that would be built in owned rooms. That way you could query what dwarf owns what buildings (e.g. cabinet, bet, modded-hobby-workshop) and issue jobs that would consume resources, give nothing back, but still level skills. Intended to work as a "do this while dwarf is idle".